docs/versioning.md

```diff
@@ -0,0 +1,46 @@
+# Versioning
+
+
+The airo-mono repo uses a single global version using a `YYYY.MM.N version scheme`.
```

Comment:
Because we're starting at 2024.1.0, could we ever go back (also on pypi) to "lower" versions such as 1.0.0, 2.5.0? Could the higher 2024 interfere with that? 

Even if so, this is not a problem for me, then we stick with date-based versions.

Reply:
will leave this as is, don't want to overcomplate for possbile future issues